src/main/views/service-results.njk

```diff
@@ -0,0 +1,62 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "macros/error-summary.njk" import errorSummary %}
+
+{% extends "template.njk" %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: '/search-option'
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row court-tribunal-details">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">
+      {{ question }}
+    </h1>
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-grid-row">
+        <div>
+          <p class="govuk-body-m">
+            {{ hint }}
+          </p>
+        </div>
+        {% for court in results.serviceAreaCourts %}
+        {% if court.catchmentType === 'national' %}
```

Comment:
seems like there would only ever be 1 national court, maybe do this in the controller if this is the case.